staramr/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# base image
+FROM ubuntu:bionic
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="staramr"
+LABEL software.version="0.5.1"
+LABEL description="staramr scans bacterial genome contigs for AMR genes and plasmids"
+LABEL website="https://github.com/phac-nml/staramr"
+LABEL license="https://github.com/phac-nml/staramr"
+
+# Maintainer
+MAINTAINER Rachael St. Jacques <rachael.stjacques@dgs.virginia.gov>
```

Comment:
This is a fairly recent change, but Docker is deprecating the MAINTAINER dockerfile instruction. Please change it to a normal LABEL like so: `LABEL maintainer="Rachael St. Jacques"` and `LABEL maintainer.email="rachael.stjacques@dgs.virginia.gov"`

Reply:
See here for an example: https://github.com/StaPH-B/docker-builds/blob/master/abricate/0.8.13/Dockerfile